driver_cpl/cime_config/config_component.xml

```diff
@@ -409,7 +409,7 @@
   <entry id="DOUT_S"> 
     <type>logical</type>
     <valid_values>TRUE,FALSE</valid_values>
-    <default_value>TRUE</default_value>
+    <default_value>FALSE</default_value>
```

Comment:
Can we leave this default set to TRUE?   We set it to FALSE for testing but generally try to have the default configuration